lib/api/objectDelete.js

```diff
@@ -171,6 +174,8 @@ function objectDelete(authInfo, request, log, cb) {
                 keys: [objectKey],
                 byteLength: Number.parseInt(objectMD['content-length'], 10),
                 numberOfObjects: 1 });
+            monitoring.promMetrics('DELETE', bucketName, '200', 'deleteObject',
```

Comment:
objectDelete vs deleteObject

Reply:
following naming convention of pushMetrics.

Reply:
Following naming convention of pushMetrics